header.php

```diff
@@ -7,12 +7,19 @@
 	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
 	<link rel="stylesheet" href="https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css">
 	<link rel="stylesheet" href="https://unpkg.com/@creativecommons/vocabulary@2020.11.3/css/vocabulary.css">
+	<script src="https://unpkg.com/vue@next"></script>
+	<script src="https://unpkg.com/@creativecommons/cc-global-components@0.2.1/dist/cc-globals.min.js"></script>
```

Comment:
Let's use semantic version placeholders here so we always get the latest minor version.
```suggestion
	<script src="https://unpkg.com/@creativecommons/cc-global-components@0.x.x/dist/cc-globals.min.js"></script>
```